pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -240,7 +240,7 @@ func wrapReports(reportList []datastreams.FeedReport, eventID string, timestamp
 
 	// Create a new CapabilityResponse with the MercuryTriggerEvent
 	return capabilities.CapabilityResponse{
-		Value: eventVal,
+		Value: eventVal.(*values.Map),
```

Comment:
can this panic? in other places you explicitly check the type coercion. why not here?

Reply:
No it can't panic: triggerEvent is a struct, and a struct is guaranteed to be wrapped as a map